SeleniumDemoQA/SeleniumDemoQA/WebDriverExtensions.cs

```diff
@@ -87,6 +87,27 @@ public static void WaitForElementEnabled(this IWebDriver driver, By by)
             wait.Until(d => d.FindElement(by).Displayed && d.FindElement(by).Enabled);
         }
 
+        public static void ClickOnTableActionElement(this IWebDriver driver, By selector)
```

Comment:
this method has nothing "Table" related. I would name ClickWithJs

Reply:
Thanks, done..)